addons/delete-others-option/userscript.js

```diff
@@ -0,0 +1,63 @@
+var addonGlobal;
```

Comment:
you can put all your functions into the `export default async funtion` and then they'll have access to `addon` already

Reply:
There should be only one export default function.

Reply:
i mean
```js
import ... from "./utils.js"
export default async ({ addon, console }) => {
  function deleteOtherCostumes(contextMenu) { /* ... */ }
  function addDeleteOthersOption(contextMenu) { /* ... */ }
  document.addEventListener("contextmenu", (e) => { /* ... */ });
};
```

Reply:
fixed